### PR TITLE
Add Endpoint For Status Check

### PR DESCRIPTION
**Pull request by sohak22**
Create A Status Endpoint
https://www.notion.so/uwblueprintexecs/Create-a-status-endpoint-eec29cc4976e48e2bfb010e50887d65d

We want to create a **/status** endpoint that returns information that the service is running. 

We want it to return 200 response code and the following message:

```json
{"status": "Healthy!"}

```

Steps to test

1. Run the application in local host : http://localhost:3000/api/status 
2. Page should return healthy status

### CODE REVIEW

File: pages/api/status.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```
Comment (feng-eric) [inline pages/api/status.tsx]: I think you can remove this line

Comment (mahadzaryab1) [inline pages/api/status.tsx]: don't need this

File: pages/api/status.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default (_req: NextApiRequest, res: NextApiResponse): void => {
```
Comment (mahadzaryab1) [inline pages/api/status.tsx:4]: add comment above function explaining what the function does

**Review by mahadzaryab1 — CHANGES_REQUESTED** [review]
Added a couple of comments - some other things to consider:
- You can delete the /hello endpoint as it's replaced by this one 
- PR name should be imperative and descriptive. For example: **_Add Endpoint For Status Check_**

**Review by feng-eric — CHANGES_REQUESTED** [review]

**Review by feng-eric — APPROVED** [review]
LGTM!

**Review by mahadzaryab1 — CHANGES_REQUESTED** [review]
I added some comments in my last review - could you take a look at those?

**Review by mahadzaryab1 — APPROVED** [review]
LGTM!

**Pull request merged**